.travis.yml

```diff
@@ -1,8 +1,11 @@
 language: node_js
-env: TRAVIS_CI=true
+env:
+  - TRAVIS_CI=true
+  - DISABLE_NOTIFIER=true
```

Comment:
I think you must specify both variables in one line. See the [travis docs](https://docs.travis-ci.com/user/environment-variables/#Defining-Multiple-Variables-per-Item):

```yaml
env:
  - TRAVIS_CI=true DISABLE_NOTIFIER=true
```